### PR TITLE
Naver

### PR DESCRIPTION
**Pull request by SlowLife1661**
# Naver - New Presence

## Naver
image
image

### Naver Papago
image
image

### Naver TV
image
image

### Naver NOW
image
image
image

### Naver Webtoon
image
image

### Naver Blog
image

### CODE REVIEW

File: websites/N/Naver/presence.ts [hunk 1]
```diff
@@ -0,0 +1,496 @@
+const presence = new Presence({
+  clientId: "825307070584586250",
+  injectOnComplete: true
```
Comment (ririxidev) [inline websites/N/Naver/presence.ts:3]: Delete it, presence is injecting no mater what after loading website.

File: websites/N/Naver/presence.ts [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+    presence: null
+  },
+
+  getLanguage = async (langCode: string) => {
+    if (!langCode) return "Choosing";
+
+    if (langCode === "auto") return "Detecting";
+    if (data.papagoLangs.has(langCode)) return data.papagoLangs.get(langCode).name;
+
+    const baseURL = "https://restcountries.eu/rest/v2/lang",
```
Comment (ririxidev) [inline websites/N/Naver/presence.ts:86]: Why? You know that you need to explain why you use this api because for now, I'm not accepting this thing.

**Review by ririxidev — CHANGES_REQUESTED** [review]
Heya @SlowLife1661!
You should check out thing(s) that I requested to change in my review. Please look at them!
 - Also squash commits.

Thanks, 
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721

File: websites/N/Naver/presence.ts (hunk 2 above)
Comment (SlowLife1661) [reply websites/N/Naver/presence.ts:86]: Reee I'm not gonna bother explain lmao, fine I will use DOM

**Comment by ririxidev** [comment]
Like i said, squash commits. (https://medium.com/@mittalyashu/how-to-squash-commits-in-a-github-pull-request-97b36576eacb)

File: websites/N/Naver/presence.ts (hunk 2 above)
Comment (ririxidev) [reply websites/N/Naver/presence.ts]: Idk I just wnated to know lol but if not K then

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by ririxidev — APPROVED** [review]
image

**Review by Bas950 — APPROVED** [review]
Lgtm

**Pull request merged**